pyproject.toml

```diff
@@ -17,6 +17,8 @@
   urllib3 =  ">=2.2.0,<3.0.0"
   websocket-client = "1.7.0"
   pieces_os_client = "1.2.7"
+  prompt-toolkit = "^3.0.43"
+rich = "^13.7.1"
```

Comment:
double check formatting here please

Reply:
Well I wrote `poetry add rich` and indentation doesn't matter in the pyproject.toml I did not added it manually